invokeai/app/cli/commands.py

```diff
@@ -72,7 +72,7 @@ def add_parsers(
 def add_graph_parsers(
     subparsers,
     graphs: list[LibraryGraph],
-    add_arguments: Callable[[argparse.ArgumentParser], None]|None = None
+    add_arguments: Union[Callable[[argparse.ArgumentParser], None], None] = None
```

Comment:
```suggestion
    add_arguments: Optional[Callable[[argparse.ArgumentParser], None]] = None
```
Note: need import Optional from typing